pkg/slo-controller/config/config.go

```diff
@@ -36,7 +36,7 @@ type ColocationCfg struct {
 
 type NodeColocationCfg struct {
 	NodeSelector *metav1.LabelSelector
-	ColocationCfg
+	ColocationStrategy
```

Comment:
Why change this api？

Reply:
Because instead of `ColocationCfg`, NodeColocationCfg shoud contains `ColocationStrategy`.